entity-framework/core/providers/index.md

```diff
@@ -16,33 +16,27 @@ Entity Framework Core can access many different databases through plug-in librar
-| [IBM.EntityFrameworkCore](https://www.nuget.org/packages/IBM.EntityFrameworkCore)      | Db2, Informix                   | [IBM](https://ibm.com)                                                                          | Paid, Windows                              | 6.0               | [getting started](https://community.ibm.com/community/user/hybriddatamanagement/blogs/michelle-betbadal1/2020/04/29/getting-started-with-ibm-net-provider-for-net-core)                                                           |
-| [IBM.EntityFrameworkCore-lnx](https://www.nuget.org/packages/IBM.EntityFrameworkCore-lnx)  | Db2, Informix                   | [IBM](https://ibm.com)                                                                          | Paid, Linux                                | 6.0               | [getting started](https://community.ibm.com/community/user/hybriddatamanagement/blogs/michelle-betbadal1/2020/04/29/getting-started-with-ibm-net-provider-for-net-core)                                                           |
-| [IBM.EntityFrameworkCore-osx](https://www.nuget.org/packages/IBM.EntityFrameworkCore-osx)  | Db2, Informix                   | [IBM](https://ibm.com)                                                                          | Paid, macOS                                | 6.0               | [getting started](https://community.ibm.com/community/user/hybriddatamanagement/blogs/michelle-betbadal1/2020/04/29/getting-started-with-ibm-net-provider-for-net-core)                                                           |
-| [EntityFrameworkCore.Jet](https://www.nuget.org/packages/EntityFrameworkCore.Jet/)                                                                                                    | Microsoft Access files          | [Bubi](https://github.com/bubibubi)                                                             |                                            | 3.1               | [readme](https://github.com/bubibubi/EntityFrameworkCore.Jet/blob/master/docs/README.md)                                                       |
-| [Teradata.EntityFrameworkCore](https://www.nuget.org/packages/Teradata.EntityFrameworkCore/)                                                                                          | Teradata Database 16.10 onwards | [Teradata](https://downloads.teradata.com/download/connectivity/net-data-provider-for-teradata) |                                            | 3.1               | [website](https://www.nuget.org/packages/Teradata.EntityFrameworkCore/)                                                                        |
```

Comment:
As with tools/extensions, maybe we should leave old providers... Like someone may need to access an old SQL Server Compact database, the information may help them even if the EF version is unsupported.